service/src/main/java/com/myhome/domain/User.java

```diff
@@ -61,4 +72,7 @@
   private String encryptedPassword;
   @ManyToMany(mappedBy = "admins", fetch = FetchType.LAZY)
   private Set<Community> communities = new HashSet<>();
+  @OneToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
```

Comment:
We can have many security tokens for single user (expired ones, different types)